### ISSUE TITLE
Crash on upload

### ISSUE THREAD

**Issue by BoBeR182** [mjaakko/NeoStumbler#27]
```
FATAL EXCEPTION: k6.d0 Dispatcher
Process: xyz.malkki.neostumbler, PID: 10421
java.lang.IllegalArgumentException: Infinity is not a valid double value as per JSON specification. To override this behavior, use GsonBuilder.serializeSpecialFloatingPointValues() method.
	at com.google.gson.j.a(Unknown Source:32)
	at com.google.gson.Gson$1.b(Unknown Source:12)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.b(Unknown Source:61)
	at com.google.gson.internal.bind.e.a(Unknown Source:86)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.b(Unknown Source:31)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.b(Unknown Source:61)
	at com.google.gson.internal.bind.e.a(Unknown Source:86)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.b(Unknown Source:31)
	at com.google.gson.internal.bind.ObjectTypeAdapter.b(Unknown Source:35)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.b(Unknown Source:61)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.b(Unknown Source:27)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.b(Unknown Source:61)
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.b(Unknown Source:55)
	at com.google.gson.j.f(Unknown Source:28)
	at com.google.gson.j.e(Unknown Source:8)
	at y7.b.v0(Unknown Source:35)
	at p6.b.a(Unknown Source:153)
	at p6.f.b(Unknown Source:121)
	at o6.a.a(Unknown Source:94)
	at p6.f.b(Unknown Source:121)
	at m6.b.a(Unknown Source:984)
	at p6.f.b(Unknown Source:121)
	at p6.a.a(Unknown Source:141)
	at p6.f.b(Unknown Source:121)
	at p6.g.a(Unknown Source:142)
	at p6.f.b(Unknown Source:121)
	at x6.a.a(Unknown Source:279)
	at p6.f.b(Unknown Source:121)
	at h8.k.a(Unknown Source:17)
	at p6.f.b(Unknown Source:121)
	at o6.j.f(Unknown Source:99)
	at o6.g.run(Unknown Source:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by mjaakko** [comment]
This should be fixed in the latest release, 1.1.1

**Comment by BoBeR182** [comment]
Fixed in 1.1.1
